examples/defaulter-gen/generators/defaulter.go

```diff
@@ -38,9 +39,54 @@ type CustomArgs struct {
 	ExtraPeerDirs []string // Always consider these as last-ditch possibilities for conversions.
 }
 
+type typeInfo struct {
+	name   string
+	format string
+	zero   interface{}
+}
+
+var schemaTypeFormatMap = map[string]typeInfo{
```

Comment:
Feel free to drop the type above and make this directly a map of zero-values.
```suggestion
var schemaTypeFormatMap = map[string]interface{}{
```
And then, because that's the only thing in the map, I think it's perfectly fine to also remove the function to access it.

Reply:
Ahh, shouldn't have copied over the function without reading everything. Will update this to just the zero-values map.